personalize.admin.inc

```diff
@@ -85,6 +85,89 @@ function personalize_admin_form_submit($form, &$form_state) {
 }
 
 /**
+ * Form for changing the status of an agent.
```

Comment:
I'm having a hard time understanding the new functions and their relation to the older _personalize_status_toggle_next and personalize_status_toggle_form_definition functions.  Seems like the new ones should replace them and then we'd just continue to refine one set of status change functions.